### PR TITLE
Tracing c# instrumentation

### PR DESCRIPTION
**Pull request by yyyogev**
Added doc for how to instrument tracing on C# app
Split doc into auto and manual instrumentation pages: 

https://deploy-preview-935--logz-docs.netlify.app/shipping/  -- see the tabs here.

https://deploy-preview-935--logz-docs.netlify.app/shipping/tracing-sources/cs-open-tracing-auto.html
https://deploy-preview-935--logz-docs.netlify.app/shipping/tracing-sources/cs-open-tracing-manual.html

----

## Pages to review

- LinkToPage1
 https://deploy-preview-935--logz-docs.netlify.app/shipping/tracing-sources/cs-open-tracing.html

## Remaining work

- [ ] Technical review
- [ ] Copy Review
- [ ] Redirects: 

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

File: _source/logzio_collections/_tracing-sources/cs_open_tracing.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+## OpenTracing and Jaeger
+
+OpenTracing is an API standard of instrumentation for distributed tracing.
+Jaeger is one implementation of that standard and our most recommended practice for C#. It has a set of libraries and tools to provide open source implementation 
+for distributed tracing.
+
+## OpenTracing for C\#
+
+### Requirements
+Add the following packages to your solution:
```
Comment (YajB) [inline _source/logzio_collections/_tracing-sources/cs_open_tracing.md:41]: @yyyogev - where is user supposed to get these packages from?

File: _source/logzio_collections/_tracing-sources/cs_open_tracing.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+* OpenTracing.Contrib.NetCore
+
+* OpenTracing
+
+* Jaeger
+
+
+### Implementation
+Add to you `Stratup.cs` file the following code snippet:
```
Comment (YajB) [inline _source/logzio_collections/_tracing-sources/cs_open_tracing.md:51]: where is the file located? Do we need to provide navigation guidance for this?

File: _source/logzio_collections/_tracing-sources/cs_open_tracing.md (hunk 1 above)
Comment (yyyogev) [reply _source/logzio_collections/_tracing-sources/cs_open_tracing.md:41]: There are several ways, the most common way is via NuGet

File: _source/logzio_collections/_tracing-sources/cs_open_tracing.md (hunk 2 above)
Comment (yyyogev) [reply _source/logzio_collections/_tracing-sources/cs_open_tracing.md:53]: Yes, should be capitalized.
No need, it's basic and it's in the root.

**Comment by yyyogev** [comment]
@YajB now I think we should move the example block to the overview tab. Since the example doesn't refer to k8s, it's confusing this way..

File: _source/logzio_collections/_tracing-sources/cs_open_tracing.md (hunk 1 above)
Comment (YajB) [reply _source/logzio_collections/_tracing-sources/cs_open_tracing.md:41]: @yyyogev  -Apologies. My intent was not clear:

 Is it reasonable for us to assume that users will know where to get the packages from or should we give them explicit guidance on where to obtain the packages?

Comment (yyyogev) [reply _source/logzio_collections/_tracing-sources/cs_open_tracing.md:41]: Nah, it's a known knowledge

**Comment by YajB** [comment]
> @YajB now I think we should move the example block to the overview tab. Since the example doesn't refer to k8s, it's confusing this way..

DONE!

**Review by yyyogev — COMMENTED** [review]
LGTM!

**Review by YajB — APPROVED** [review]

**Pull request merged**